src/bundle/Resources/config/routing.yml

```
@@ -479,6 +479,14 @@ ibexa.rest.create_location:
 
 # Content types
 
+ibexa.rest.content_type.copy:
+    path: /content/types/{contentTypeId}
+    defaults:
+        _controller: Ibexa\Rest\Server\Controller\ContentType:copyContentType
+    condition: 'is_compatible_content_type(request, "application/vnd.ibexa.api.CopyContentTypeInput")'
```

I would personally rename it to:
```suggestion
    condition: 'is_content_type_compatible(request, "application/vnd.ibexa.api.CopyContentTypeInput")'
```
but since naming is a bit biased, I would let others give their 5 cents.

---

Agreed, sounds much better.